src/Microsoft.TemplateEngine.Abstractions/TemplatePackage/IManagedTemplatePackageProvider.cs

```diff
@@ -44,7 +44,7 @@ public interface IManagedTemplatePackageProvider : ITemplatePackageProvider
         /// <summary>
         /// Installs new <see cref="IManagedTemplatePackage"/> based on <see cref="InstallRequest"/> data.
         /// All <see cref="IInstaller"/>s are considered via <see cref="IInstaller.CanInstallAsync(InstallRequest, CancellationToken)"/> and if only 1 <see cref="IInstaller"/>
-        /// returns <c>true</c>. <see cref="IInstaller.InstallAsync(InstallRequest, CancellationToken)"/> is executed and result is returned.
+        /// returns <c>true</c>. <see cref="IInstaller.InstallAsync(InstallRequest, IManagedTemplatePackageProvider, CancellationToken)"/> is executed and result is returned.
```

Comment:
Super nit: `<see langword="true" />` is probably a better fit than `<c>true</c>` here.
```suggestion
        /// returns <see langword="true" />. <see cref="IInstaller.InstallAsync(InstallRequest, IManagedTemplatePackageProvider, CancellationToken)"/> is executed and result is returned.
```